src/main/com/fulcrologic/rad/form_options.cljc

```diff
@@ -21,6 +21,10 @@
   isn't in this list, it will not be managed."
   :com.fulcrologic.rad.form/attributes)
 
+(def read-only-fields
+  "OPTIONAL: Render this set of fields as read-only."
```

Comment:
I appreciate the help on docstrings, but in this case I'm not sure I'm keeping that option long-term, so I skipped it. For future reference the description needs to be of book quality since these are the primary docs. The description you're giving is ambiguous (i.e. "a set of qualified keys" instead of "fields").

Reply:
Also, as a note, you need to remember to look how the option is used. Many will allow a lambda, in which case you also have to document that, and what options it is passed...this is part of the reason I'm waiting on this option. Should this be a lambda that takes the form instance, a qkey, and returns true/false?  Probably, because many forms will need dynamic control over read-only

Reply:
Sounds good, I'll remove this.